pytissueoptics/scene/tree/spacePartition.py

```diff
@@ -105,3 +115,24 @@ def getLeafBoundingBoxesAsCuboids(self) -> List[Cuboid]:
             c = bbox.zMax - bbox.zMin
             cuboids.append(Cuboid(a=a, b=b, c=c, position=bbox.center))
         return cuboids
+
+    def getJSONBranching(self, node: Node = None, jsonFile: list = None):
+        if jsonFile is None:
+            jsonFile = []
+
+        if node is not None:
+            for i, child in enumerate(node.children):
+                jsonFile.append({"depth": child.depth, "polygons": len(child.polygons), "bbox": f"{child.bbox:.2f}", "children": []})
+                self.getJSONBranching(child, jsonFile[i]["children"])
+
+        else:
+            node = self._root
+            jsonFile.append({})
+            jsonFile[0]["depth"] = node.depth
+            jsonFile[0]["size"] = len(node.polygons)
+            jsonFile[0]["bbox"] = f"{node.bbox:.2f}"
+            jsonFile[0]["children"] = []
+            self.getJSONBranching(node, jsonFile[0]["children"])
+
+        if node.isRoot:
+            return json.dumps(jsonFile)
```

Comment:
I understand this is for debugging. 
If you really want to keep it, put it private or outside package maybe.

Reply:
did put it outside the package. resolved with commit 61f6dd15c9719c1995e0c7f4bad10dc3969e3686